types/testing-library__jest-dom/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for @testing-library/jest-dom 5.9
+// Type definitions for @testing-library/jest-dom 5.13
 // Project: https://github.com/testing-library/jest-dom
 // Definitions by: Ernesto García <https://github.com/gnapse>
 //                 John Gozde <https://github.com/jgoz>
 //                 Seth Macpherson <https://github.com/smacpherson64>
+//                 Shenghao Lei <https://github.com/SevenOutman>
```

Comment:
To confirm, this list isn't for attribution - the git history handles that - it's for people who want to be notified when these types change so that they can provide their feedback on the updates.

Reply:
@SevenOutman you may want to take yourself off of this list then.

Reply:
I see. Going to remove this line then.